src/Selenium2Library/locators/elementfinder.py

```diff
@@ -192,6 +194,8 @@ def _element_matches(self, element, tag, constraints):
         if not element.tag_name.lower() == tag:
             return False
         for name in constraints:
+            if type(constraints[name]) is list and element.get_attribute(name) not in constraints[name]:
+                return False
```

Comment:
> Why is this change needed?

Currently constraints only accept 1 value per element attribute and the change involves the creation of a list of values that should be considered acceptable.

> Should use usinstance, not tyoe

Will do the change. Never coded for Python so I really don't know what's best. :) Thanks for the review! 👍